src/analyzer/analyzersilence.h

```diff
@@ -13,15 +14,29 @@ class AnalyzerSilence : public Analyzer {
     bool initialize(TrackPointer pTrack,
             mixxx::audio::SampleRate sampleRate,
             int totalSamples) override;
-    bool processSamples(const CSAMPLE* pIn, const int iLen) override;
+    bool processSamples(const CSAMPLE* pIn, SINT iLen) override;
     void storeResults(TrackPointer pTrack) override;
     void cleanup() override;
 
+    /// returns the index of the first sample in the buffer that is above -60 dB
+    /// or iLen if no sample is found
+    static SINT findFirstSoundInChunk(const CSAMPLE* pIn, SINT iLen);
+
+    /// returns the index of the last sample in the buffer that is above -60 dB
+    /// or -1 if no sample is found. signalStart can be set to a known sample above
```

Comment:
Comment is outdated.

Reply:
The comment is correct, the code was wrong. I will amend the last commit.

Reply:
The parameter signalStart does not exist.

Reply:
Oh, right.